workspaces/ui-v2/src/optic-components/pages/diffs/CapturePage.tsx

```diff
@@ -24,19 +28,29 @@ import {
   useDiffUndocumentedUrlsPageLink,
 } from '../../navigation/Routes';
 import { EndpointName } from '../../common';
-import { useHistory } from 'react-router-dom';
 import ApproveAll from '../../diffs/render/ApproveAll';
 import { useCaptures } from '../../hooks/useCapturesHook';
 
-export function CapturePage(props: { showDiff?: boolean }) {
+function CapturePage(props: { showDiff?: boolean }) {
+  const { boundaryId, environment } = useParams<{
+    boundaryId?: string;
+    environment?: string;
+  }>();
   const capturesState = useCaptures();
   const history = useHistory();
   const diffEnvironmentsRoot = useDiffEnvironmentsRoot();
 
-  const noCaptures =
-    !capturesState.loading && capturesState.captures.length === 0;
+  const noCaptures = capturesState.captures.length === 0;
+  // The only current environment supported is local
+  const isValidEnvironment = environment === 'local';
```

Comment:
I'm not sure if this is correct - but this is what I've found everywhere in the UI at least? And I can't see where else this is set

Reply:
yeah, it'll change when we move the other envs into the OSS project, but this is correct today